src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepositoryTest.java

```diff
@@ -9,15 +10,17 @@
 import uk.gov.hmcts.reform.notificationservice.model.common.ErrorCode;
 
 import java.util.Collections;
+import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.tuple;
 import static uk.gov.hmcts.reform.notificationservice.data.NotificationStatus.PENDING;
 
 @SpringBootTest
 public class NotificationRepositoryTest {
 
-    @Autowired NamedParameterJdbcTemplate jdbcTemplate;
-    @Autowired NotificationRepository notificationRepository;
+    @Autowired private NamedParameterJdbcTemplate jdbcTemplate;
+    @Autowired private NotificationRepository notificationRepository;
```

Comment:
can be reverted

Reply:
why not to make it private?

Reply:
Same reason test methods don't have access modifier - it can be omitted.
See router project for reference.